src/assets/locales/fr.json

```
@@ -0,0 +1,567 @@
+  "media-browser.add_custom_avatar": "Avatar GLB URL",
+  "media-browser.add_custom_object": "Custom URL or File",
+  "media-browser.add_custom_scene": "Custom Scene",
+  "media-browser.create-avatar": "Créer un avatar",
+  "media-browser.create-scene": "Créer une scène avec {editorName}",
+  "media-browser.empty.default": "Aucun résultat. Essayez de faire une nouvelle recherche.",
+  "media-browser.empty.favorites": "Vous n'avez pas de favoris. Cliquer sur ⭐ pour ajouter à vos favoris.",
+  "media-browser.favorites-header": "Salles favorites",
+  "media-browser.nav_title.avatars": "Avatars",
+  "media-browser.nav_title.gifs": "GIFs",
+  "media-browser.nav_title.images": "Images",
+  "media-browser.nav_title.poly": "Google Poly",
+  "media-browser.nav_title.scenes": "Scenes",
+  "media-browser.nav_title.sketchfab": "Sketchfab",
+  "media-browser.nav_title.twitch": "Twitch",
+  "media-browser.nav_title.videos": "Videos",
+  "media-browser.nav_title.youtube": "YouTube",
+  "media-browser.powered_by.gifs": "Chercher via Tenor",
+  "media-browser.powered_by.images": "Chercher via Bing",
+  "media-browser.powered_by.poly": "Chercher via Google",
+  "media-browser.powered_by.scenes": "Fait avec {editorName}",
+  "media-browser.powered_by.sketchfab": "Chercher via Sketchfab",
+  "media-browser.powered_by.twitch": "Chercher via Twitch",
+  "media-browser.powered_by.videos": "Chercher via Bing",
+  "media-browser.powered_by.youtube": "Chercher via Google",
+  "media-browser.privacy_policy": "Politique de confidentialité",
+  "media-browser.report-issue": "Signaler une erreur",
+  "media-browser.search-placeholder.avatars": "Chercher des Avatars...",
+  "media-browser.search-placeholder.default": "Chercher...",
+  "media-browser.search-placeholder.gifs": "Chercher des GIFs...",
+  "media-browser.search-placeholder.images": "Chercher des Images...",
+  "media-browser.search-placeholder.poly": "Chercher des modèles Google Poly...",
+  "media-browser.search-placeholder.scenes": "Chercher des Scènes...",
+  "media-browser.search-placeholder.sketchfab": "Chercher des modèles Sketchfab...",
+  "media-browser.search-placeholder.twitch": "Chercher des streams Twitch...",
+  "media-browser.search-placeholder.videos": "Chercher des Videos...",
+  "media-browser.search-placeholder.youtube": "Chercher des videos Youtube...",
+  "media-tile.action.copy-avatar": "Copier vers mes avatars",
+  "media-tile.action.copy-scene": "Copier vers mes scenes",
+  "media-tile.action.edit-avatar": "Éditer l'avatar",
+  "media-tile.action.edit-scene": "Éditer la scène",
+  "media-tile.action.room-info": "Info de la Salle",
+  "media-tile.action.show-similar-avatars": "Montrer les avatars similaires",
+  "media-tile.joined-room": "Joint {relativeTime}",
+  "mic-permissions-modal.grant-mic-permissions": "Accorder les autorisations au Microphone",
```

```suggestion
  "mic-permissions-modal.grant-mic-permissions": "Accorder les autorisations au micro",
```
@ssm2017 is it possible for you to modify all the "Microphone" occurences to "micro" ?